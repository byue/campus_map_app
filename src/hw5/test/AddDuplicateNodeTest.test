# Tests that duplicate nodes are not inserted into the graph

# Create a graph
CreateGraph graph1

# Add 5 nodes including 2 duplicates
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n1
AddNode graph1 n2

# list nodes of graph
ListNodes graph1