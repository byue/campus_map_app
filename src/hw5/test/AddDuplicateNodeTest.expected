# Tests that duplicate nodes are not inserted into the graph

# Create a graph
created graph graph1

# Add 5 nodes including 2 duplicates
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
graph1 already contains node n1
graph1 already contains node n2

# list nodes of graph
graph1 contains: n1 n2 n3